config/_shared.json

```diff
@@ -4,6 +4,7 @@
 	"favicon_url": "//s1.wp.com/i/favicon.ico",
 	"boom_analytics_enabled": false,
 	"server_side_boom_analytics_enabled": false,
+	"server_side_memcached": false,
```

Comment:
Is that necessary? Isn't it disabled by default unless we specifically enable it in an environment config file?

Reply:
That's what I thought, but I actually got an error during compilation that said the key didn't exist, which was fixed by adding it here. I think that behavior may be true for "features", but not for this. (I'm sure why "features" are different from other keys in the config file.)